src/private/__tests__/actions.test.js

```diff
@@ -393,4 +398,24 @@ describe('actions', () => {
       })
     })
   })
+
+  describe('resetState', () => {
+    it('Resets state', () => {
+      const getResult = actions.getRequest(store, {
+        operation: 'list',
+        name: 'get',
+        stateId: 'list',
+      })
+
+      expect(getResult).toBeTruthy()
+      expect(get(store.state, 'get@list')).toMatchObject({
+        fetchingList: true,
+        listError: null,
+      })
+
+      actions.resetState(store)
+
+      expect(store.state).toEqual({})
+    })
+  })
```

Comment:
I'm pretty sure this test (together with lines 14 to 18) isn't testing the correct behaviour. What am I getting wrong?